### PR TITLE
Python 3 compatibility

### PR DESCRIPTION
**Pull request by taldcroft**
Passing tests except for test_aca_hdr3.py.

Got through most of the docs examples OK.  These should just be put into regression tests where sensible.

### CODE REVIEW

File: mica/archive/aca_l0.py [hunk 1]
```diff
@@ -206,11 +207,13 @@ def __init__(self, msids, start, stop):
...
-    aca_db = dict(dbi="sybase", server="sybase", user="aca_read")
-    with Ska.DBI.DBI(**aca_db) as db:
-        obspars = db.fetchall("""select * from obspar where obsid = %d
-                                 order by obi""" % obsid)
-    return obspars[0]['tstart'], obspars[0]['tstop']
+    from kadi import events  # Kadi is a big import so defer
+    dwells = events.dwells.filter(obsid=obsid)
+    n_dwells = len(dwells)
+    tstart = dwells[0].tstart
+    tstop = dwells[n_dwells - 1].tstop
```
Comment (jeanconn) [inline mica/archive/aca_l0.py]: Might want to also handle non-"contiguous" dwells with a warning or some such.

Comment (jeanconn) [inline mica/archive/aca_l0.py]: I don't recall what obsid_times is doing anyway.  Seems like it should have been reading from the mica obspar table if it really cared about obspars.

Comment (taldcroft) [inline mica/archive/aca_l0.py]: I think most such warnings would be false alarms from small discontinuities like monitor window commanding.  In any case functional enhancements are mostly out of scope for this PR.

From my reading `obsid_times` is a one-off internal method to get times for aca_l0.  It's only called from one place.  So I think the sybase => kadi replacement code is fine.

Comment (jeanconn) [inline mica/archive/aca_l0.py]: Yes.  (Though I didn't mean real multi-dwell obsids as much as the set that kadi might return for obsid=0)

Comment (taldcroft) [inline mica/archive/aca_l0.py]: One needs to find a balance between possible corner cases and progress.  :smile:

File: mica/starcheck/starcheck_parser.py [hunk 2]
```diff
@@ -5,11 +5,12 @@
 import re
 import subprocess
 import json
-from itertools import izip, count
+from itertools import count
 import numpy as np
 import asciitable
```
Comment (jeanconn) [inline mica/starcheck/starcheck_parser.py]: This missing in /proj/sot/ska3/test

**Comment by taldcroft** [comment]
Are these warnings new?  They show up in ska_testr (which runs tests so that warnings show up).  Not a blocker, but I'm wondering if this will generate alerts in ska_watch.
```
mica/vv/tests/test_vv.py::test_run_vv Generating V&V for obsid 2121
Processing aspect interval pcadf090549491N003
WARNING: hdu= was not specified but multiple tables are present, reading in first available table (hdu=1) [astropy.io.fits.connect]
WARNING: UnitsWarning: 'marcsec' did not parse as fits unit: At col 0, Unit 'marcsec' not supported by the FITS standard. Did you mean arcsec? [astropy.units.core]
WARNING: UnitsWarning: 'marcsec/yr' did not parse as fits unit: At col 0, Unit 'marcsec' not supported by the FITS standard. Did you mean arcsec? [astropy.units.core]
```

**Comment by jeanconn** [comment]
Yes, those are from the change from Ska.Table to astropy.table.  I hadn't decided yet about muting them in mica.vv.core.

**Comment by taldcroft** [comment]
We can deal with it later...

**Comment by taldcroft** [comment]
BTW the `hdu` one is trivial, just include it in the call.  I'm not sure if there is an easy way to disable the unit warnings.

**Comment by taldcroft** [comment]
See https://github.com/astropy/astropy/pull/5675, which if accepted will fix the warnings problem.

**Comment by jeanconn** [comment]
@taldcroft Are you planning to merge and tag this PR?  I assume there are others hanging around but I wasn't clear on your process plan.

**Comment by taldcroft** [comment]
My plan is to merge all py3 branches now along with py2-3 for skare.  I might try to find a way to scriptify the tagging.

**Pull request merged**